src/content-blocks/BlockFlowPlayer/BlockFlowPlayer.tsx

```diff
@@ -16,4 +17,10 @@ export interface BlockFlowPlayerProps extends DefaultProps {
 export const BlockFlowPlayer: FunctionComponent<BlockFlowPlayerProps> = ({
 	className,
 	...flowPlayerProps
-}) => <FlowPlayer className={className} {...flowPlayerProps} />;
+}) => (
+	<FlowPlayer
+		className={className}
+		{...flowPlayerProps}
+		autoplay={isNil(flowPlayerProps.autoplay) ? true : flowPlayerProps.autoplay}
```

Comment:
Deze eventueel gwn in uw function params defaulten naar true als deze undefined is?